pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -330,6 +335,13 @@ type operationExecutor struct {
 	// pendingOperations keeps track of pending attach and detach operations so
 	// multiple operations are not started on the same volume
 	pendingOperations nestedpendingoperations.NestedPendingOperations
+
+	// recorder is used to record events in the API server
+	recorder record.EventRecorder
+
+	// represents a volume that should be attached to this node and
+	// mounted to the PodName.
+	volumeToMount VolumeToMount
```

Comment:
I don't understand why this was added here. And I don't see where it is populated.